fixtures/gists-app/pnpm-lock.yaml

```diff
@@ -0,0 +1,4853 @@
+lockfileVersion: 5.3
```

Comment:
@Industrial I'm going to nudge some other folks on the team to review, but in the mean time do you mind removing this file from the PR? I added `pnpm` lockfiles to `gitignore` on `dev` so this shouldn't be an issue if you merge.